### PR TITLE
[DOCS] More fixes for asciidoctor migration

### PR DESCRIPTION
**Pull request by dedemorton**
Fixes asciidoc coding issues that break the asciidoctor build.

Remaining to do:

- [x] Make sure asciidoc builds continue to work (will be tested by CI on this PR)
- [x] ~Compare asciidoc build output to asciidoctor build output to see what's different and make changes, as required.~ removed because this testing needs to happen after all the changes are merged and backported.

### CODE REVIEW

File: libbeat/scripts/generate_fields_docs.py [hunk 1]
```diff
@@ -73,9 +73,11 @@ def document_field(output, field, field_path):
             output.write("{}\n\n".format("Object is not enabled."))
```
Comment (andrewkroh) [inline libbeat/scripts/generate_fields_docs.py]: Would it work the same if you just moved the original `output.write` line to here?

Comment (dedemorton) [reply libbeat/scripts/generate_fields_docs.py]: Yes, that works! Thanks for helping me simplify the logic. I'm good at getting things to work...but not so good at keeping it simple and elegant. :-)

**Comment by dedemorton** [comment]
I think this is ready to merge. I don't think there is an easy way to test the asciidoctor builds without backporting to all branches first, but I did compare the HTML before/after to make sure the output is correct in the asciidoc build.

**Comment by dedemorton** [comment]
jenkins test this again please

**Pull request merged**